aws/PAS-network-environment-NAT.json

```diff
@@ -1026,6 +1069,392 @@
+                }
+            }
+        },
+        "PTASGIngress4": {
+            "Type": "AWS::EC2::SecurityGroupIngress",
+            "Properties": {
+                "GroupId": {
+                    "Ref": "PTASG"
+                },
+                "IpProtocol": "tcp",
+                "FromPort": 8443,
+                "ToPort": 8443,
+                "CidrIp": {
+                    "Ref": "AdministrativeAccessCIDR"
+                }
+            }
+        },
+        "PTASGIngress5": {
+            "Type": "AWS::EC2::SecurityGroupIngress",
+            "Properties": {
+                "GroupId": {
+                    "Ref": "PTASG"
+                },
+                "IpProtocol": "udp",
+                "FromPort": 67,
+                "ToPort": 68,
+                "CidrIp": "0.0.0.0/0"
+            }
+        },
+        "PTASGIngress6": {
+            "Type": "AWS::EC2::SecurityGroupIngress",
+            "Properties": {
+                "GroupId": {
+                    "Ref": "PTASG"
+                },
+                "IpProtocol": "tcp",
+                "FromPort": 27017,
+                "ToPort": 27017,
+                "DestinationSecurityGroupId": {
+                    "Ref": "PTASG"
+                }
+            }
+        },
+        "PTASGIngress7": {
+            "Type": "AWS::EC2::SecurityGroupIngress",
+            "Properties": {
+                "GroupId": {
+                    "Ref": "PTASG"
+                },
+                "IpProtocol": "tcp",
+                "FromPort": 6514,
+                "ToPort": 6514,
+                "CidrIp": "0.0.0.0/0"
+            }
+        },
+        "PTASGIngress8": {
+            "Type": "AWS::EC2::SecurityGroupIngress",
+            "Properties": {
+                "GroupId": {
+                    "Ref": "PTASG"
+                },
+                "IpProtocol": "tcp",
+                "FromPort": 7514,
+                "ToPort": 7514,
+                "CidrIp": "0.0.0.0/0"
+            }
+        },
+        "PTASGIngress9": {
+            "Type": "AWS::EC2::SecurityGroupIngress",
+            "Properties": {
+                "GroupId": {
+                    "Ref": "PTASG"
+                },
+                "IpProtocol": "tcp",
+                "FromPort": 22,
+                "ToPort": 22,
+                "DestinationSecurityGroupId": {
+                    "Ref": "PTASG"
+                }
+            }
+        },
+        "PTASGIngress10": {
+            "Type": "AWS::EC2::SecurityGroupIngress",
+            "Properties": {
+                "GroupId": {
+                    "Ref": "PTASG"
+                },
+                "IpProtocol": "tcp",
+                "FromPort": 22,
+                "ToPort": 22,
+                "CidrIp": {
+                    "Ref": "AdministrativeAccessCIDR"
+                }
+            }
+        },
+        "PTASGInress12": {
+            "Type": "AWS::EC2::SecurityGroupEgress",
```

Comment:
Shouldn't it be AWS::EC2::SecurityGroupIngress if it is defined under PTASGInress12?